apm/otel_monitor.go

```diff
@@ -136,7 +141,13 @@ func (m *monitor) Started(ctx context.Context, evt *event.CommandStartedEvent) {
 	}
 	if !m.cfg.CommandAttributeDisabled {
 		if stmt := m.cfg.CommandTransformerFunc(evt.Command); stmt != "" {
-			attrs = append(attrs, semconv.DBStatement(stmt))
```

Comment:
so before we were dropping this in directly, and now we're doing a formatted and stripped version; will we not need this unprocessed version anymore?  (I'm not really sure what the difference is between it and the formatted version)

Reply:
The unprocessed version contains additional driver cruft and isn't pretty printed. You can see an example [here](https://ui.honeycomb.io/mongodb-4b/environments/production/datasets/evergreen/result/GRPF1EgQGWf/trace/geebAkuyWqK?fields[]=s_name&fields[]=s_serviceName&span=14349f17849c70e7). The new version has just the query and is pretty printed for easier human consumption.